Src/value.h

```diff
@@ -10,7 +10,7 @@
 #define SEEN_VALUE_H
 
 /* Types of primitives held in Value type*/
-typedef enum {VALUE_FLOAT, VALUE_INT} ValueKind;
+typedef enum {VALUE_INT, VALUE_FLOAT} ValueKind;
```

Comment:
I am curious about the reasons for this change. Does it mask some issues, e.g., when Kind is not initialized/initialized to 0?

Reply:
Yep - historically, this change was done to make a default initialization of values in tokens as int  - otherwise, in some cases I was getting floats by default (e.g. if we have var declared in promela without assignment, spin creates a token node in the ini attribute of the tree with zero value...). I think, this change is not necessary anymore, but I forgot about it as the tests are going fine right now

Reply:
In this case, I would remove this change, at least for now - it might be usefull to discover issues with initialization. If at the end we decide that these issues cannot be resolved by the book, we will reconsider adding it.